module/main.go

```diff
@@ -0,0 +1,39 @@
+// Package main provides an ORB_SLAM3 implementation of a SLAM module
+package main
+
+import (
+	"context"
+
+	"github.com/edaniels/golog"
+	"go.viam.com/rdk/module"
+	"go.viam.com/rdk/services/slam"
+	"go.viam.com/utils"
+
+	viamorbslam3 "github.com/viamrobotics/viam-orb-slam3"
+)
+
+func main() {
+	utils.ContextualMain(mainWithArgs, golog.NewLogger("orbslam3Module"))
+}
+
+func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error {
+	// Instantiate the module
+	orbModule, err := module.NewModuleFromArgs(ctx, logger)
+	if err != nil {
+		return err
+	}
+
+	// Add model to the module
+	if err = orbModule.AddModelFromRegistry(ctx, slam.Subtype, viamorbslam3.Model); err != nil {
+		return err
```

Comment:
Under what conditions will this error?

Reply:
If the subtype & model are are not within the regestry: i.e. if `viamorbslam3.init()` doesn't register `slam.Subtype` and  `viamorbslam3.Model` properly